modules/dkan/dkan_data_dictionary/js/editor.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @file
+ * Provides options for chart visualization.
+ */
+
+(function ($) {
```

Comment:
Even though it works, this code could surely be better. My javascript is very rusty